charts/sda-svc/README.md

```diff
@@ -135,13 +135,26 @@ Parameter | Description | Default
 `global.inbox.nfsPath` | Path on the NFS server for the inbox. |`""`
 `global.inbox.existingClaim` | Existing volume to use for the `posix` inbox. | `""`
 `global.inbox.s3Url` | URL to S3 inbox instance. |`""`
+`global.inbox.s3Port` | Port that the S3 inbox is available on. |`443`
 `global.inbox.s3Bucket` | S3 inbox bucket. |`""`
 `global.inbox.s3Region` | S3 inbox region. |`""`
 `global.inbox.s3ChunkSize` | S3 chunk size in MB. |`15`
 `global.inbox.s3AccessKey` | Access key to S3 inbox . |`null`
 `global.inbox.s3SecretKey` | Secret key to S3 inbox. |`null`
 `global.inbox.s3CaFile` | CA certificate to use if the S3 inbox is internal. |`null`
 `global.inbox.s3ReadyPath` | Endpoint to verify that the inbox is respondig. |`""`
+`global.sync.api.password` | Password for authenticating to the syncAPI server | `null`
+`global.sync.api.user` | User for authenticating to the syncAPI server | `null`
+`global.sync.centerPrefix` | Prefix for locally generated datasets | `null`
+`global.sync.destination.storageType` | Storage type for the sync destination, currently only supports S3 | `s3`
```

Comment:
Why do we default other `storageType`s  to `null` but this to `s3`?

Reply:
Because the current implementation only works with S3